cassandra_template.yaml

```diff
@@ -539,7 +539,7 @@ batch_size_warn_threshold_in_kb: 5
 #
 # If your data directories are backed by SSD, you should increase this
 # to the number of cores.
```

Comment:
Given that compaction of compressed SSTables (which is the case by default) is CPU-bound, that doesn't seem to be a particularly good advise.